docs/sources/datasources/prometheus/_index.md

```diff
@@ -175,3 +177,11 @@ Grafana lists these variables in dropdown select boxes at the top of the dashboa
 Grafana refers to such variables as template variables.
 
 For details, see the [template variables documentation]({{< relref "./template-variables/" >}}).
+
+## Incremental Dashboard Queries (beta)
+
+As of Grafana 10, data sources can be configured to query live dashboards incrementally, instead of re-querying the entire duration on each dashboard refresh.
```

Comment:
```suggestion
As of Grafana 10, the Prometheus data source can be configured to query live dashboards incrementally, instead of re-querying the entire duration on each dashboard refresh.
```